install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -11,7 +11,7 @@
 {{- $defaultBpfCtAnyMax := 262144 -}}
 {{- $enableIdentityMark := "true" -}}
 {{- $fragmentTracking := "true" -}}
-{{- $defaultKubeProxyReplacement := "false" -}}
+{{- $defaultKubeProxyReplacement := (toString "false") -}}
```

Comment:
This `toString` here should not be needed. `"false"` is already a string. (most other uses of `toString` are fine in this PR, just this particular instance here is superfluous)

Reply:
You are correct, this `toString` is not needed, took it a bit overboard. Will remove.